docs/admin/integrations/platformx.md

```diff
@@ -12,17 +12,59 @@ their experience.
 
 You'll need:
 
-- A running Coder v2.19+ deployment
+- Coder v2.19+
 - A PlatformX subscription from [DX](https://getdx.com/)
-- A platform to host the integration (choose any of these):
+- A platform to host the integration, such as:
   - AWS Lambda
+  - Google Cloud Run
```

Comment:
added this based on the [readme](https://github.com/coder/coder-platformx-notifications?tab=readme-ov-file#deployment)

> This middleware is optimized for serverless environments such as Google Cloud Run and AWS Lambda.